kvm-install-vm

```diff
@@ -392,6 +393,14 @@ function fetch_images ()
             DISK_FORMAT=qcow2
             LOGIN_USER=ubuntu
             ;;
+        opensuse15)
+            QCOW=openSUSE-Leap-15.2-OpenStack.x86_64.qcow2
+            IMAGE_URL=https://download.opensuse.org/repositories/Cloud:/Images:/Leap_15.2/images
```

Comment:
I just noticed the ordering of these values is not consistent with the other cases. I can fix that if you like. Also, why are some of the values quoted and some are bare strings in this long case statement?

Reply:
I pushed an update that fixes the ordering. Thanks.